### PR TITLE
Adds support using a custom domain to bypass ad blockers

### PR DESCRIPTION
**Pull request by khrome83**
I added this based on the documentation here.

https://docs.simpleanalytics.com/bypass-ad-blockers

### What this PR does

1. Adds support for a optional custom domain
2. Defaults the custom domain property to the correct `cdn.simpleanalytics.io`
3. Generate a new build in `/dist` with the changes
4. Updates the readme to show the new functionality, including linking to offical docs
5. Updates the package version to `1.1.0` since this expands functionality

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -6,29 +6,29 @@ const warn = message => {
...
   el.async = true;
-  el.src = 'https://cdn.simpleanalytics.io/hello.js';
+  el.src = 'https://' + domain + '/hello.js';
   document.head.appendChild(el);
 }
 
 export default {
-  install(vue, { skip } = { skip: false }) {
-    if (skip === false) return injectScript()
+  install(vue, { skip = false, domain = 'cdn.simpleanalytics.io' }) {
```
Comment (adriaandotcom) [inline src/index.js:21]: Cool, didn't know about this syntax.

File: src/index.js [hunk 2]
```diff
@@ -6,29 +6,29 @@ const warn = message => {
...
 }
 
-const injectScript = () => {
+const injectScript = (domain) => {
   if (!document) return warn('No document defined.');
   const el = document.createElement('script');
   el.type = 'text/javascript';
   el.async = true;
-  el.src = 'https://cdn.simpleanalytics.io/hello.js';
+  el.src = 'https://' + domain + '/hello.js';
```
Comment (adriaandotcom) [inline src/index.js]: We are only using `/app.js` for custom domains in the future. Could you update this? So `/hello.js` for `cdn.simpleanalytics.io` and `/app.js` for any other domain.

**Review by adriaandotcom — CHANGES_REQUESTED** [review]
Thanks for this PR, really appreciated! There is one thing that needs an update.

File: src/index.js (hunk 2 above)
Comment (khrome83) [reply src/index.js]: Sure. I will have this done before I go to work.

**Review by khrome83 — COMMENTED** [review]
This should resolve the `app.js` vs `hello.js` issue.

**Review by adriaandotcom — APPROVED** [review]

**Comment by adriaandotcom** [comment]
Published to npm: https://www.npmjs.com/package/simple-analytics-vue/v/1.1.0

Thank you!

**Pull request merged**